src/components/ebay-textbox/README.md

```diff
@@ -15,6 +15,8 @@ Name | Type | Stateful | Description
 `fluid` | Boolean | No |
 `multiline` | Boolean | No | renders a multi-line texbox if true
 `invalid` | Boolean | No | indicates a field-level error with red border if true
+`icon-name` | String | No | name of the icon from skin
```

Comment:
This is now simply `icon` and not `icon-name`.

Reply:
Done :)